# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/UseCounters.conf
# Commit: ebbd2cd3905e
# Full Hash: ebbd2cd3905e3542c417ea33e0c2cdd88260e6c9
# Author: Cristina Horotan <chorotan@mozilla.com>
# Date: 2025-05-22 09:08:48
# Regressor Bug: 1964451
# File Overlap Count: 1
# Description:
#   Revert "Bug 1964451 - Add use counters for OOM handling r=sfink,mccr8" on request
#   
#   This reverts commit 7ae56a029252266ce967356e2254b1408b8522ea.
# ==============================================================================

diff -r 34bb3734b886 -r ebbd2cd3905e dom/base/UseCounters.conf
--- a/dom/base/UseCounters.conf	Wed May 21 18:52:10 2025 +0000
+++ b/dom/base/UseCounters.conf	Wed May 21 22:38:12 2025 +0300
@@ -85,10 +85,6 @@
 custom JS_dateparse called Date.parse
 custom JS_dateparse_impl_def call to Date.parse used implementation defined behaviour
 custom JS_regexp_symbol_protocol_on_primitive passed a primitive with a custom implementation of the regexp protocol symbols to one of the String methods that accept a regexp
-custom JS_large_oom_reported reported a large oom
-custom JS_small_oom_reported reported a small oom
-custom JS_large_oom_recovered recovered a large oom
-custom JS_small_oom_recovered recovered a small oom
 
 // Console API
 method console.assert